backend/shork/src/main/kotlin/internal/compiler/Token.kt

```diff
@@ -0,0 +1,29 @@
+package software.shonk.interpreter.internal.compiler
+
+internal class Token(val type: TokenType, val lexeme: String, val literal: Any, val line: Int) {
```

Comment:
Describe clearly what data this Token class contains. What is the lexeme and especially what is the literal?

Reply:
Explaining domain knowledge should not be the purpose of a comment, unless maybe if it is very specific